### PR TITLE
Make SonarQube pick up the Cobertura Code Coverage file

### PR DESCRIPTION
**Pull request by bgavrilMS**
Note that SQ seems to be able to pick up the Jacoco report without help, so not doing anything there.

### CODE REVIEW

File: Tasks/Gradle/gradlesonar.ts [hunk 1]
```diff
@@ -27,8 +27,8 @@ export function applyEnabledSonarQubeArguments(gradleRun: trm.ToolRunner):trm.To
     var initScriptPath:string = path.join(__dirname, 'sonar.gradle');
 
     // Specify that the build should run the init script
-    gradleRun.arg(['-I', initScriptPath]);
     gradleRun.arg(['sonarqube']);
+    gradleRun.arg(['-I', initScriptPath]);
```
Comment (bgavrilMS) [inline Tasks/Gradle/gradlesonar.ts]: Small unrelated change - I'm adding the "sonarqube" argument before the -I arg so that all the task arguments are grouped

Comment (bgavrilMS) [inline Tasks/Gradle/gradlesonar.ts]: @AshwiniChalla - just as an aside, we found that you can pass init Gradle scripts that contain plugins. This way we didn't have to change the user's gradle.build file.

File: Tasks/Gradle/gradletask.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ var gb = tl.createToolRunner(wrapperScript);
 
 gb.argString(tl.getInput('options', false));
 gb.arg(tl.getDelimitedInput('tasks', ' ', true));
-gb = sqGradle.applyEnabledSonarQubeArguments(gb);
+
```
Comment (bgavrilMS) [inline Tasks/Gradle/gradletask.ts]: I moved the SQ logic after the CC logic

File: Tasks/Gradle/sonar.gradle [hunk 3]
```diff
@@ -16,11 +16,6 @@ allprojects {
         if (project.getParent() == null && !project.plugins.hasPlugin('org.sonarqube')) {
             //Add sonar plugin only to root projects, and only if it is not already applied
             project.apply plugin: 'org.sonarqube'
-        }
-
-        // Warn if the project name has not been set
-        if (project.name.equalsIgnoreCase("s")) {
```
Comment (bgavrilMS) [inline Tasks/Gradle/sonar.gradle]: Unrelated change - this is not needed as we now ask for a SQ project name

File: Tasks/Gradle/task.json [hunk 4]
```diff
@@ -210,7 +210,7 @@
             "type": "string",
             "label": "SonarQube Project Name",
             "required": true,
-            "helpMarkDown": "The SonarQube project name, i.e. sonar.projectName. If not set, the plugin will select its own default. See: http://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner+for+Gradle",
```
Comment (bgavrilMS) [inline Tasks/Gradle/task.json]: Unrelated change - the plugin will not set anything by default because the project name, key and version are mandatory.

**Comment by HSAR** [comment]
LGTM
